src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -1,6 +1,6 @@
+/* eslint-disable no-unexpected-multiline */
```

Comment:
Had to disable this eslint rule temporarily because it was giving errors locally when running a build.